liblocation/src/main/java/com/mapbox/android/core/location/GoogleLocationEngine.java

```diff
@@ -134,14 +133,14 @@ public void requestLocationUpdates() {
 
     if (googleApiClient.isConnected()) {
```

Comment:
Hey @electrostat .
Great work on this one! Not sure if this pull request should also already fix https://github.com/mapbox/mapbox-events-android/issues/163 as well, but unless you set LocationRequest.setMaxWaitTime(long) you will not receive location updates in a batched mode. (https://developers.google.com/android/reference/com/google/android/gms/location/LocationRequest.html#setMaxWaitTime(long))

Cheers

Reply:
Hey @florianPOLARSTEPS 👋 this PR is aiming to unlock the work for #163, where we will then complete the `Location` batching setup.  Thanks for the quick feedback here and stay tuned as we get work going on that PR.